package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "main-magicians",
+  "name": "math-magicians",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "big-js": "^3.1.3",
     "big.js": "^6.1.1",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.0.0",
```

Comment:
[OPTIONAL] When I run your application, I get a compile error.
![image](https://user-images.githubusercontent.com/38088818/173183473-69abd778-4c8c-4a22-93db-073de4687cad.png)

* The reason is that you imported calculator.css with a capital letter "c" in calculator.js component. 
    
![image](https://user-images.githubusercontent.com/38088818/173183577-c4f22f81-189d-40e2-8689-bc186bcfa4b8.png)

- Kindly look into this to make sure that your application works properly.